styles/style.css

```diff
@@ -0,0 +1,855 @@
+  display: inline-block;
+  margin-top: 5px;
+}
+
+.pick-side-info a:hover {
+  text-decoration: none;
+  color: #f92859;
+}
+
+.article-category :nth-child(2) a {
+  padding: 0;
+}
+
+.mag-article-content h3 {
+  font-size: 14px;
+  margin-bottom: 30px;
+  font-weight: 700;
+  color: black;
+}
+
+.last-art-img-category h3 {
+  position: absolute;
+  bottom: 3px;
+  color: rgba(255, 255, 255, 0.9);
+  padding: 8px 10px 8px 10px;
+  background-color: #f72210;
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+.last-mag-discription {
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+  margin: 10px 0 30px;
+}
+
+.subscribe-header h2 {
+  color: #f72210;
+  font-size: 16px;
+  font-weight: 700;
+  padding-bottom: 25px;
+  text-transform: uppercase;
+}
+
+.subscribe-section {
+  flex-direction: column;
+}
+
+.subscribe-image-container {
+  position: relative;
+  height: 160px;
+  margin: 0 auto 15px;
+  display: inline-block;
+  align-self: center;
+  padding-bottom: 13px;
+  max-width: 290px;
+}
+
+.subscribe-img-top,
+.subscribe-img-bottom {
+  width: 170px;
+  height: 228px;
+}
+
+.subscribe-img-top {
+  position: absolute;
+  top: 19px;
+  left: 84px;
+}
+
+.subscribe-button {
+  background-color: #f72310;
+  color: white;
+  text-transform: uppercase;
+  padding: 10px;
+  font-weight: 700;
+  font-size: 16px;
+  border-radius: 3px;
+  cursor: pointer;
+  border: none;
+}
+
+.subscribe-title {
+  align-self: center;
+  text-align: center;
+  width: 90%;
+  z-index: 1;
+}
+
+.subscribe-title h3 {
+  font-size: 18px;
+  color: #000;
+  font-weight: 700;
+  margin-bottom: 20px;
+}
+
+.subscribe-title p {
+  line-height: 18px;
+  margin-bottom: 20px;
+  color: #333;
+  font-size: 13px;
+}
+
+.news-letter-info {
+  flex-direction: column;
+  align-items: center;
+}
+
+.news-letter-sign-up-title {
+  font-size: 19px;
+  font-weight: 700;
+  margin-bottom: 15px;
+}
+
+.sign-up-info {
+  color: #333;
+  font-size: 13px;
+  margin-bottom: 20px;
+  font-weight: 700;
+  text-align: center;
+  line-height: 18px;
+}
+
+.sign-up-newsletter {
+  max-width: 100%;
+  margin-bottom: 30px;
+  border: 1px solid #e5e5e5;
+  background-color: #f4f5f6;
+  padding: 15px;
+}
+
+.social-footer,
+.footer-logo {
+  flex-direction: column;
+}
+
+.social-footer {
+  border-bottom: 1px solid rgba(3, 3, 3, 0.12);
+  margin-bottom: 10px;
+}
+
+.footer-logo svg {
+  width: 236px;
+  fill: #f72210;
+  margin: 0 15px 12px 0;
+}
+
+.copyright {
+  font-size: 12px;
+  color: #666;
+  margin-bottom: 12px;
+  text-transform: uppercase;
+}
+
+.social-icons i {
+  font-size: 23px;
+  color: #666;
+  margin: 0 10px 10px 0;
+}
+
+.info-footer {
+  font-size: 14px;
+  color: #666;
+  margin-bottom: 30px;
+}
+
+.sign-up-img {
+  width: 48%;
+  max-width: 191px;
+  background-image: url(../images/starting5-homepage.png);
+  background-size: cover;
+  background-position: center;
+}
+
+.subscribe-img {
+  justify-content: center;
+}
```

Comment:
This could also be achieved with bootstrap. ([Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options))